tests/test_data_source.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+import io
+import data.source
+
+
+def test_get_from_csv():
+    f = io.StringIO("""﻿question,answer
+我要查曠課,昌昌;曠課
+我要查課表,昌昌;課表
+我要查成績,昌昌;成績
```

Comment:
Restrict your command type to enum; I know it's very attractive to using command text as the original definition but even that is a detail for this prototype so that's unnecessary and not good

Reply:
I think we should not to define any command_type or intents. Keep this pure and judge the similarity by the inputs.

Reply:
But what you do is implicit import command type which doesn't match what you say

Reply:
It's just a text example with CSV format.

Reply:
The code would explain itself; the test code is no exception. That's why we should not do that.